projects/fsastorefrontlib/src/cms-components/my-account/policy/policy-details/policy-details.component.html

```diff
@@ -1,32 +1,22 @@
 <ng-container *ngIf="policy$ | async as policy">
   <div class="overview-section mb-5">
-    <span
-      *ngIf="policy.mainProduct && policy.mainProduct.coverageProduct"
-      class="image-wrapper"
+    <a
```

Comment:
In consultation with designer, removed excess code, to sync the styling with other pages